src/mon/PGMap.cc

```diff
@@ -4550,8 +4556,41 @@ int reweight::by_utilization(
       osd_util.second = (double)p.second.kb_used / (double)p.second.kb;
     }
     util_by_osd.push_back(osd_util);
+
+    // Overweighted devices and their weights are factors to
```

Comment:
@amitkumar50 could you please explain the rationale behind it?  i asked the same question at https://github.com/ceph/ceph/pull/16715#discussion_r132932242, but no response.